docs/_documentations/mdt-vsc-troubleshooting.md

```diff
@@ -18,6 +18,18 @@ You can also post questions on our [Mattermost channel](https://mattermost.eclip
 
 ***
 
+### **Codewind Output stream**
+
+The Codewind Output stream is a tab within Codewind that is visible in the VS Code editor. It logs `cwctl` commands together with their ouput, for example:
```

Comment:
> is a tab within Codewind that is visible in the VS Code editor.

It is not a tab, and it is not "within Codewind". The vs code term _is_ output stream. So I think

`The Codewind Output stream is now available in the VS Code editor.`

or similar, is sufficient.

Reply:
Reworded.